docs/design/2022-09-19-distributed-ddl-reorg.md

```diff
@@ -0,0 +1,310 @@
+# Proposal: Distributed DDL Reorg
+
+- Author(s): [zimulala](https://github.com/zimulala), [Defined2014](https://github.com/Defined2014)
+- Tracking Issue: https://github.com/pingcap/tidb/issues/41208
+
+# <a id="_37pie9l9cba0"></a>__Abstract__
+
+This is distributed processing of design in the DDL reorg phase.
```

Comment:
I do not think here need use two paragraphs
“The document proposes a design for the distributed processing of DDL reorg phase in TiDB. The current design is based on the main logic that only the DDL owner can handle DDL jobs. However, this proposal suggests that all TiDBs should be able to claim subtasks in the reorg phase based on resource usage.”